cmd/utils/nodecmd/dumpconfigcmd.go

```diff
@@ -120,22 +146,652 @@ func makeConfigNode(ctx *cli.Context) (*node.Node, klayConfig) {
+		params.BlockGenerationInterval = ctx.GlobalInt64(utils.BlockGenerationIntervalFlag.Name)
+		if params.BlockGenerationInterval < 1 {
+			logger.Crit("Block generation interval should be equal or larger than 1", "interval", params.BlockGenerationInterval)
+		}
+	}
+	if ctx.GlobalIsSet(utils.BlockGenerationTimeLimitFlag.Name) {
+		params.BlockGenerationTimeLimit = ctx.GlobalDuration(utils.BlockGenerationTimeLimitFlag.Name)
+	}
+
+	params.OpcodeComputationCostLimit = ctx.GlobalUint64(utils.OpcodeComputationCostLimitFlag.Name)
+
+	if ctx.GlobalIsSet(utils.SnapshotFlag.Name) {
+		cfg.SnapshotCacheSize = ctx.GlobalInt(utils.SnapshotCacheSizeFlag.Name)
+		if cfg.StartBlockNumber != 0 {
+			logger.Crit("State snapshot should not be used with --start-block-num", "num", cfg.StartBlockNumber)
+		}
+		logger.Info("State snapshot is enabled", "cache-size (MB)", cfg.SnapshotCacheSize)
+	} else {
+		cfg.SnapshotCacheSize = 0 // snapshot disabled
+	}
+
+	// Override any default configs for hard coded network.
+	// TODO-Klaytn-Bootnode: Discuss and add `baobab` test network's genesis block
+	/*
+		if ctx.GlobalBool(TestnetFlag.Name) {
+			if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+				cfg.NetworkId = 3
+			}
+			cfg.Genesis = blockchain.DefaultBaobabGenesisBlock()
+		}
+	*/
+	// Set the Tx resending related configuration variables
+	setTxResendConfig(ctx, cfg)
+}
+
+// raiseFDLimit increases the file descriptor limit to process's maximum value
+func raiseFDLimit() {
+	limit, err := fdlimit.Maximum()
+	if err != nil {
+		logger.Error("Failed to read maximum fd. you may suffer fd exhaustion", "err", err)
+		return
+	}
+	raised, err := fdlimit.Raise(uint64(limit))
+	if err != nil {
+		logger.Warn("Failed to increase fd limit. you may suffer fd exhaustion", "err", err)
+		return
+	}
+	logger.Info("Raised fd limit to process's maximum value", "fd", raised)
+}
+
+// setServiceChainSigner retrieves the service chain signer either from the directly specified
+// command line flags or from the keystore if CLI indexed.
+func setServiceChainSigner(ctx *cli.Context, ks *keystore.KeyStore, cfg *cn.Config) {
+	if ctx.GlobalIsSet(utils.ServiceChainSignerFlag.Name) {
+		account, err := makeAddress(ks, ctx.GlobalString(utils.ServiceChainSignerFlag.Name))
+		if err != nil {
+			log.Fatalf("Option %q: %v", utils.ServiceChainSignerFlag.Name, err)
+		}
+		cfg.ServiceChainSigner = account.Address
+	}
+}
+
+// setRewardbase retrieves the rewardbase either from the directly specified
+// command line flags or from the keystore if CLI indexed.
+func setRewardbase(ctx *cli.Context, ks *keystore.KeyStore, cfg *cn.Config) {
+	if ctx.GlobalIsSet(utils.RewardbaseFlag.Name) {
+		account, err := makeAddress(ks, ctx.GlobalString(utils.RewardbaseFlag.Name))
+		if err != nil {
+			log.Fatalf("Option %q: %v", utils.RewardbaseFlag.Name, err)
+		}
+		cfg.Rewardbase = account.Address
+	}
+}
+
+// MakeAddress converts an account specified directly as a hex encoded string or
```

Comment:
```suggestion
// makeAddress converts an account specified directly as a hex encoded string or
```
Seems like a minor typo.